renovate.json

```diff
@@ -1,10 +1,18 @@
 {
-  "ignoreDeps": ["dvc", "execa", "@types/vscode"],
+  "ignoreDeps": ["execa", "@types/vscode"],
   "extends": ["config:base"],
   "packageRules": [
+    {
+      "description": "Automatically merge non-major updates",
+      "automerge": true,
+      "automergeType": "branch",
+      "matchDatasources": ["npm"],
+      "matchUpdateTypes": ["minor", "patch", "digest", "pin"]
+    },
     {
       "internalChecksFilter": "strict",
-      "matchDatasources": ["npm", "pypi"],
+      "matchDatasources": ["npm"],
+      "prConcurrentLimit": 1,
```

Comment:
[F] This should limit the number of dependency updates to 1. This will get rid of the waste of rebasing all of the PRs all the time.

Reply:
the limit excludes security updates